Doc/library/logging.rst

```diff
@@ -873,21 +873,21 @@ wire).
 
    :param args: Variable data to merge into the *msg* argument
       to obtain the event description.
-   :type args: tuple | dict[str, typing.Any]
+   :type args: tuple or dict[str, typing.Any]
 
    :param exc_info: An exception tuple with the current exception information,
       as returned by :func:`sys.exc_info`,
       or ``None`` if no exception information is available.
-   :type exc_info: tuple[type[BaseException], BaseException, types.TracebackType] | None
+   :type exc_info: tuple[type[BaseException], BaseException, types.TracebackType] or None
```

Comment:
Is this one still perhaps a bit too complex for the docs? Not sure how it could be simplified, though

Reply:
It is definitely complex. In this case, I'd say readability is definitely improved using `or` iso. `|`.